src/modules/commander/commander_params.c

```diff
@@ -87,58 +87,73 @@ PARAM_DEFINE_FLOAT(TRIM_PITCH, 0.0f);
 PARAM_DEFINE_FLOAT(TRIM_YAW, 0.0f);
 
 /**
- * Empty cell voltage.
+ * Full battery voltage.
  *
- * Defines the voltage where a single cell of the battery is considered empty.
+ * Defines the voltage at which the battery is considered full.
  *
  * @group Battery Calibration
  * @unit V
+ * @min 0.0
  */
-PARAM_DEFINE_FLOAT(BAT_V_EMPTY, 3.4f);
+PARAM_DEFINE_FLOAT(BAT_V_FULL, 12.6f);
 
 /**
- * Full cell voltage.
+ * Low battery voltage.
  *
- * Defines the voltage where a single cell of the battery is considered full.
+ * Defines the voltage at which the battery is considered low.
  *
  * @group Battery Calibration
  * @unit V
+ * @min 0.0
  */
-PARAM_DEFINE_FLOAT(BAT_V_CHARGED, 4.2f);
+PARAM_DEFINE_FLOAT(BAT_V_LOW, 10.8f);
 
 /**
- * Voltage drop per cell on 100% load
+ * Critical battery voltage.
  *
- * This implicitely defines the internal resistance
- * to maximum current ratio and assumes linearity.
+ * Defines the voltage at which the battery is considered critically empty.
  *
  * @group Battery Calibration
  * @unit V
- * @min 0.0f
+ * @min 0.0
  */
-PARAM_DEFINE_FLOAT(BAT_V_LOAD_DROP, 0.07f);
+PARAM_DEFINE_FLOAT(BAT_V_CRIT, 10.2f);
 
 /**
- * Number of cells.
+ * Battery capacity.
  *
- * Defines the number of cells the attached battery consists of.
+ * Defines the capacity of the attached battery.
  *
  * @group Battery Calibration
- * @unit S
- * @min 1
- * @max 10
+ * @unit mAh
```

Comment:
Seems odd to be a float instead of an int. Also I assume -1 signals not set? If so, can you put that in the description?